editor/src/components/inspector/inspector.tsx

```diff
@@ -351,7 +350,7 @@ export const Inspector = React.memo<InspectorProps>((props: InspectorProps) => {
     'RootElementIndicator aRootElementIsSelected',
   )
 
-  const allSelecteElementsdAreChildrenOfAGroup = useEditorState(
+  const allSelectedElementsdAreChildrenOfAGroup = useEditorState(
```

Comment:
this can be deleted as it is no longer used actually! 

also there's one more typo remaining in the name haha :)

Reply:
removed in [`c940333` (#4159)](https://github.com/concrete-utopia/utopia/pull/4159/commits/c940333053d7544c5b010db6ec0b7bd9b60ca953)